### ISSUE TITLE
Unable to publish an upload to a collection

### ISSUE THREAD

**Issue by babaklar** [MESH-Research/commons-team#34]
While testing collections and trying to publish [Barty's inspirational book](https://works.hcommons.org/collections/kcspotlight/requests/b7a1cd57-e1a3-4980-a336-81d49c9c70e9), I made a change to the metadata where I added [Larissa Babak](https://hcommons.org/members/babaklar/) as a collaborator. After going through the process I went to the "Save and Publish" page. I selected "publish to collection" and received this warning message. 

<img width="1360" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMWhXQnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--0cf45b2adaaf22e5e46baaad2dbfad0534858d89/Screenshot%202024-10-17%20at%201.47.14%E2%80%AFPM.png" alt="Screenshot 2024-10-17 at 1.47.14 PM.png" />

After checking the boxes and hitting the publish button, I received a message saying "please correct the errors in this form." I clicked through the form and didn't see any errors. 

<img width="1315" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMWxXQnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--4bb3d292837613ea604e9055a5692c559469329f/Screenshot%202024-10-17%20at%201.42.34%E2%80%AFPM.png" alt="Screenshot 2024-10-17 at 1.42.34 PM.png" />

**Comment by babaklar** [comment]
I went back through the steps I wrote here yesterday with the same results. Then I went back through the upload form one more time to look for error messages. There isn't a message, but this red X box shows up on the details page. When I click on the X, it removes that final line from this field. Now, even after I do this I still can't publish the deposit, but I'm wondering if it's something in that area. <img width="1260" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNkphQnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--990b2222f9156417bfb03f542e315f2d727c340a/Screenshot%202024-10-18%20at%201.39.06%E2%80%AFPM.png" alt="Screenshot 2024-10-18 at 1.39.06 PM.png" />
